payloads/v10/monetization.ts

```diff
@@ -0,0 +1,81 @@
+import type { Snowflake } from '../../globals';
+
+export interface APIEntitlement {
+	/**
+	 * ID of the entitlement
+	 */
+	id: Snowflake;
+	/**
+	 * ID of the SKU
+	 */
+	sku_id: Snowflake;
+	/**
+	 * ID of the user that is granted access to the entitlement's sku
+	 */
+	user_id?: Snowflake;
+	/**
+	 * ID of the guild that is granted access to the entitlement's sku
+	 */
+	guild_id?: Snowflake;
+	/**
+	 * ID of the parent application
+	 */
+	application_id: Snowflake;
+	/**
+	 * Type of entitlement
+	 */
+	type: EntitlementType;
+	/**
+	 * Not applicable for App Subscriptions. Subscriptions are not consumed and will be false
+	 */
+	consumed: boolean;
+	/**
+	 * Start date at which the entitlement is valid. Not present when using test entitlements.
+	 */
+	starts_at?: number;
+	/**
+	 * Date at which the entitlement is no longer valid. Not present when using test entitlements.
+	 */
+	ends_at?: number;
+}
+
+export enum EntitlementType {
+	/**
+	 * Entitlement was purchased as an app subscription
+	 */
+	ApplicationSubscription = 8,
+}
+
+export interface APISKU {
```

Comment:
Should have `flags` now as of https://github.com/discord/discord-api-docs/pull/6452.